src/azure/accountStore.ts

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import vscode = require('vscode');
+import { IAccount } from '../models/contracts/azure/accountInterfaces';
+import Constants = require('../constants/constants');
+import AzureAuth = require('@cssuh/ads-adal-library');
+
+export class AccountStore {
+    constructor(
+        private _context: vscode.ExtensionContext
+    ) { }
+
+    public getAccounts(): IAccount[] {
+        let configValues = this._context.globalState.get<IAccount[]>(Constants.configAzureAccount);
```

Comment:
` ?? []`

Reply:
could you clarify what you mean?

Reply:
![image](https://user-images.githubusercontent.com/1465785/93136089-33065d00-f690-11ea-9ed8-6416b73dc775.png)
you can basically replace this undefined check with `??`